test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -1296,4 +1297,26 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			assert.strictEqual( content, originalContent, 'The restored post content is not correct' );
 		} );
 	} );
+
+	describe( 'Settings @parallel', function () {
+		step( 'Can log in', async function () {
+			const loginFlow = new LoginFlow( driver, gutenbergUser );
+			await loginFlow.loginAndStartNewPost( null, true );
+		} );
+
+		step( 'Line height setting is available', async function () {
+			const editor = await GutenbergEditorComponent.Expect( driver );
+			await editor.enterText( 'Foobar' );
+
+			const gSidebarComponent = await GutenbergEditorSidebarComponent.Expect( driver );
+			await gSidebarComponent.displayComponentIfNecessary();
+			await gSidebarComponent.chooseBlockSettings();
+
+			const lineHeighSettingPresent = await driverHelper.isElementPresent(
+				driver,
+				By.css( '.block-editor-line-height-control' )
+			);
+			assert.strictEqual( lineHeighSettingPresent, true, 'Line height setting not found' );
```

Comment:
I think this line is equal to:

```js
assert.ok( lineHeightSettingPresent, 'Line height setting not found.' );
```

Just a tiny nitpick - feel free to ignore 👍

Reply:
Ah, that's much leaner, thanks for the suggestion!